packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -49,6 +49,7 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
       const popperHasScrollableParent = getScrollParent(contentRef.current) !== document.body
 
       const modifiers: PopperJS.Modifiers = _.merge(
+        { preventOverflow: { padding: 0 } },
```

Comment:
Should be this in modifiers in `Dropdown` component?

Reply:
I think it's something we want for all our components